shell/platform/android/io/flutter/embedding/engine/renderer/FlutterRenderer.java

```diff
@@ -649,6 +654,9 @@ PerImage dequeueImage() {
 
     @Override
     public void onTrimMemory(int level) {
+      if (!CLEANUP_ON_MEMORY_PRESSURE) {
+        return;
```

Comment:
Do we expect to re-enable this code path again soon? Is there another reason we want to keep it around instead of fully reverting the changes from the original PR?

Reply:
Probably not going to re-enable this code path but want the option to do that in the future :)

Reply:
Fair enough!